src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -114,6 +115,50 @@ def new_forward(*inputs):
 
         layer.module.forward = new_forward
 
+    def export_model(self, model, save_path=None, onnx_path=None, input_shape=None):
+        """
+        Export pruned model weights, masks and onnx model(optional)
+
+        Parameters
+        ----------
+        model : pytorch model
+            pytorch model to save
+        save_path : str
+            path to save state_dict and mask_dict
+        onnx_path : str
+            (optional) path to save onnx model
+        input_shape : list or tuple
+            input shape to onnx model
+        """
+        export_dict = {}
+        export_dict['state_dict'] = model.state_dict()
+        mask_dict = {}
+        for layer_name in self.mask_list:
+            mask_dict[layer_name + '.weight'] = self.mask_list[layer_name]
```

Comment:
It is a little be wield to do this conversion here, normally we do not call a mask as weight.   And if we need to do the conversion, better to save it as 'xxx.weight' at compression time.

Reply:
@chicm-ms I've fixed, using a dict to store masks, users can get mask like this
 ```mask = mask_dict[layer_name]['weight']```